{"abstract":[{"type":"text","text":"The error message represented by this instance, this string is presented to"},{"type":"text","text":" "},{"type":"text","text":"the user when a "},{"type":"codeVoice","code":"ValidationError"},{"text":" is thrown from either; ","type":"text"},{"type":"codeVoice","code":"run()"},{"type":"text","text":","},{"type":"text","text":" "},{"type":"codeVoice","code":"validate()"},{"text":" or a transform closure.","type":"text"}],"hierarchy":{"paths":[["doc:\/\/com.SwiftyLab.HelperCoders\/documentation\/ArgumentParser","doc:\/\/com.SwiftyLab.HelperCoders\/documentation\/ArgumentParser\/ValidationError"]]},"schemaVersion":{"patch":0,"minor":3,"major":0},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/argumentparser\/validationerror\/message"]}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.SwiftyLab.HelperCoders\/documentation\/ArgumentParser\/ValidationError\/message"},"sections":[],"kind":"symbol","metadata":{"modules":[{"name":"ArgumentParser"}],"roleHeading":"Instance Property","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"message","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"}],"role":"symbol","symbolKind":"property","externalID":"s:14ArgumentParser15ValidationErrorV7messageSSvp","title":"message"},"primaryContentSections":[{"declarations":[{"languages":["swift"],"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"text":"message","kind":"identifier"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"kind":"text","text":" { get }"}],"platforms":["macOS"]}],"kind":"declarations"}],"references":{"doc://com.SwiftyLab.HelperCoders/documentation/ArgumentParser/ValidationError/message":{"kind":"symbol","title":"message","identifier":"doc:\/\/com.SwiftyLab.HelperCoders\/documentation\/ArgumentParser\/ValidationError\/message","url":"\/documentation\/argumentparser\/validationerror\/message","role":"symbol","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"message"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"}],"type":"topic","abstract":[{"type":"text","text":"The error message represented by this instance, this string is presented to"},{"text":" ","type":"text"},{"type":"text","text":"the user when a "},{"code":"ValidationError","type":"codeVoice"},{"text":" is thrown from either; ","type":"text"},{"code":"run()","type":"codeVoice"},{"text":",","type":"text"},{"text":" ","type":"text"},{"type":"codeVoice","code":"validate()"},{"type":"text","text":" or a transform closure."}]},"doc://com.SwiftyLab.HelperCoders/documentation/ArgumentParser/ValidationError":{"type":"topic","url":"\/documentation\/argumentparser\/validationerror","abstract":[{"type":"text","text":"An error type that is presented to the user as an error with parsing their"},{"type":"text","text":" "},{"type":"text","text":"command-line input."}],"title":"ValidationError","kind":"symbol","navigatorTitle":[{"text":"ValidationError","kind":"identifier"}],"role":"symbol","fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"ValidationError"}],"identifier":"doc:\/\/com.SwiftyLab.HelperCoders\/documentation\/ArgumentParser\/ValidationError"},"doc://com.SwiftyLab.HelperCoders/documentation/ArgumentParser":{"title":"ArgumentParser","kind":"symbol","type":"topic","url":"\/documentation\/argumentparser","identifier":"doc:\/\/com.SwiftyLab.HelperCoders\/documentation\/ArgumentParser","abstract":[],"role":"collection"}}}